### ISSUE TITLE
how to get image pose for localization image without focal length?

### ISSUE THREAD

**Issue by hiakru** [colmap/colmap#398]
While localizing based on an existed 3D model, COLMAP seems require localization image with focal length. If not, it seems could not be successfully return the pose.
In function EstimateAbsolutePoseKernel, the feature point2D will be transfer to camera coordinate by 

>   // Normalize image coordinates with current camera hypothesis.
>   std::vector<Eigen::Vector2d> points2D_N(points2D.size());
>   for (size_t i = 0; i < points2D.size(); ++i) {
>     points2D_N[i] = scaled_camera.ImageToWorld(points2D[i]);
>   }

But what if there is no focal length there? Seems no camera model could help....
Thus, my question is how could I localize image without focal length?

Detail at https://github.com/colmap/colmap/blob/dev/src/estimators/pose.cc#L50

**Comment by hiakru** [comment]
Yes, I had tried that. but in fact focal length will be set as denominator when do ImageToWorld transformation. Focal length is still necessary.
points2D_N[i] = scaled_camera.ImageToWorld(points2D[i]);

**Comment by ahojnnes** [comment]
Set it to a reasonable value then. Something like 1.25 times the image width.
